layers/+source-control/version-control/packages.el

```diff
@@ -105,14 +106,15 @@
 
 (defun version-control/init-git-gutter+ ()
   (use-package git-gutter+
-    :commands (global-git-gutter+-mode git-gutter+-mode)
+    :commands (global-git-gutter+-mode git-gutter+-mode git-gutter+-refresh)
```

Comment:
Yes, without it, the `magit-pre-refresh-hook` below will fail if you open magit before git-gutter+ loads from the idle timer.